ci/Jenkinsfile-kubernetes

```diff
@@ -309,31 +309,116 @@ def stopExternalH2OBackend() {
+    sh 'sleep 60'
+    sh "kubectl get pod driver-r | grep -q Completed && echo \"OK\" || exit 1"
     stopExternalH2OBackend()
 }
 
 def testScalaExternalBackendClusterMode(registryId, master, version) {
     def leaderNodeIpPort = startExternalH2OBackend(registryId, version)
-    //TODO
+    sh "kubectl delete pod driver-scala --ignore-not-found"
+    sh """
+        \$SPARK_HOME/bin/spark-submit \
+         --conf spark.kubernetes.container.image=${registryId}.dkr.ecr.us-east-2.amazonaws.com/sw_kubernetes_repo/sparkling-water:scala-${version} \
+         --conf spark.kubernetes.driver.pod.name=driver-scala \
+         --conf spark.scheduler.minRegisteredResourcesRatio=1 \
+         --master $master \
+         --deploy-mode cluster \
+         --name test \
+         --conf spark.ext.h2o.backend.cluster.mode=external \
+         --conf spark.ext.h2o.external.start.mode=manual \
+         --conf spark.ext.h2o.hadoop.memory=2G \
+         --conf spark.ext.h2o.cloud.representative=$leaderNodeIpPort \
+         --conf spark.ext.h2o.cloud.name=root \
+         --class ai.h2o.sparkling.InitTest \
+         --conf spark.executor.instances=2 \
+         local:///opt/sparkling-water/tests/initTest.jar
+        """
+    sh "kubectl get pod driver-scala | grep -q Completed && echo \"OK\" || exit 1"
     stopExternalH2OBackend()
 }
 
 def testScalaExternalBackendClientMode(registryId, master, version) {
     def leaderNodeIpPort = startExternalH2OBackend(registryId, version)
-    //TODO
+    sh "kubectl delete pod sparkling-water-app --ignore-not-found"
+    installSparkHeadlessService()
+    def image = "${registryId}.dkr.ecr.us-east-2.amazonaws.com/sw_kubernetes_repo/sparkling-water:scala-${version}"
+    sh """
+        kubectl run -n default -i --tty sparkling-water-app --restart=Never --labels spark-driver-selector=sparkling-water-app \
+        --image=${image} -- \
+        \$SPARK_HOME/bin/spark-submit \
+         --conf spark.scheduler.minRegisteredResourcesRatio=1 \
+         --conf spark.kubernetes.container.image=${image}  \
+         --master $master \
+         --class ai.h2o.sparkling.InitTest \
+         --conf spark.driver.host=sparkling-water-app \
+         --deploy-mode client \
+         --conf spark.ext.h2o.backend.cluster.mode=external \
+         --conf spark.ext.h2o.external.start.mode=manual \
+         --conf spark.ext.h2o.hadoop.memory=2G \
+         --conf spark.ext.h2o.cloud.representative=$leaderNodeIpPort \
+         --conf spark.ext.h2o.cloud.name=root \
+         --conf spark.executor.instances=2 \
+         local:///opt/sparkling-water/tests/initTest.jar
+        """
+    sh "kubectl get pod sparkling-water-app | grep -q Completed && echo \"OK\" || exit 1"
     stopExternalH2OBackend()
 }
 
 def testPythonExternalBackendClusterMode(registryId, master, version) {
     def leaderNodeIpPort = startExternalH2OBackend(registryId, version)
-    //TODO
+    sh "kubectl delete pod driver-python --ignore-not-found"
```

Comment:
ditto for `testPythonExternalBackendClusterMode` and `testPythonExternalBackendClientMode`

all scala and python tests share the same `--master` and `--conf` options

Reply:
Good spot, they are duplicates. The motivation for keeping it like this was that it might be easier to read what all options are applied to specific test.

But I can extract some methods